pyat/test/conftest.py

```diff
@@ -3,9 +3,26 @@
 """
 import numpy
 import pytest
+from at import load_mat
 
 
 @pytest.fixture
 def rin():
     rin = numpy.array(numpy.zeros((6, 1)), order='F')
     return rin
+
+@pytest.fixture(scope='session')
```

Comment:
Why do you need these fixtures? Can't you just use `load_mat.CLASSES` in the tests?

Reply:
I tried it and it didn't work...I can't remember why though. I'll take another look.

Reply:
Apologies, it doesn't work with pytest.mark.parametrize or passing it as an argument without fixtures. It does however work if you just define it at the top of the file and then don't pass it as an argument to the tests. I'll commit this change as it's simpler than using fixtures.